roles/download/defaults/main.yml

```diff
@@ -49,7 +49,7 @@ download_delegate: "{% if download_localhost %}localhost{% else %}{{ groups['kub
 image_arch: "{{host_architecture | default('amd64')}}"
 
 # Versions
-kube_version: v1.19.5
```

Comment:
this definition should be removed

in another PR.

Reply:
some other variables in roles/download/defaults/main.yml are using `kube_version`, won't it break? not sure how cross-playbooks variables actually works